code/modules/vehicles/spacepod/equipment/sensors.dm

```diff
@@ -0,0 +1,78 @@
+/obj/item/pod_equipment/sensors
+	slot = POD_SLOT_SENSORS
+	name = "sensor suite"
+	desc = "A sensor suite for space pods, containing some tech doodads and a built-in GPS. Manufacted by Nanotrasen, and has weird labels like \"Not for infowar\" or \"manufacted on Tau Ceti IV\"."
+	icon_state = "sensors"
+	/// reference to our GPS component
+	var/datum/component/gps/item/gps
+	/// traits given to all passengers
+	var/traits_given
+
+/obj/item/pod_equipment/sensors/on_attach(mob/user)
+	. = ..()
+	gps = AddComponent(/datum/component/gps/item, "POD[rand(0,999)]", state = GLOB.in_vehicle_state, overlay_state = FALSE)
```

Comment:
why are we holding a reference to a component this isn't right

Reply:
we need a ref to check if it is on to drain power from the pod, to call its ui_interact (maybe this one could be just called in a different way), to close its UI once it drains of power